recipe/meta.yaml

```diff
@@ -26,9 +26,9 @@ requirements:
     - python >={{ python_min }}
     - feedparser >=6.0.11
     - pytz
-    - requests >=2.21.0
     - bibtexparser
     - httpx >=0.28.1
+    - pytest-asyncio
 
 test:
```

Comment:
```suggestion
test:
  requirements:
    - pytest-asyncio    
```

Reply:
Actually, if we are not running the test suite here, I don't think there is any need to add it.